### PR TITLE
adds movie and show slash commands

### PR DESCRIPTION
**Pull request by Pecunia201**
This change addresses #7 by adding slash commands for the 'show' and 'movie' commands.

Known issue:
When using the slash commands, on reject, the _apply function will not complete. It should work fine with normal commands.

### CODE REVIEW

File: src/bot/discord_bot.py [hunk 1]
```diff
@@ -423,18 +424,64 @@ async def _add_user(m, options):
...
 
+tree = app_commands.CommandTree(client)
 
 @client.event
 async def on_ready() -> None:
     await on_startup()
+    await tree.sync(guild=discord.Object(id=1080421430426091551))
+
+# MOVIE SLASH COMMAND
+@tree.command(name="movie", description="Request a movie", guild=discord.Object(id=1080421430426091551))
```
Comment (nickheyer) [inline src/bot/discord_bot.py:437]: I assume this guild-id is a placeholder for debugging, right?

File: src/bot/discord_bot.py [hunk 2]
```diff
@@ -423,18 +424,64 @@ async def _add_user(m, options):
...
 
-# BEGIN CLIENT EVENTS -----
+# SLASH COMMANDS
 
+tree = app_commands.CommandTree(client)
 
 @client.event
 async def on_ready() -> None:
     await on_startup()
+    await tree.sync(guild=discord.Object(id=1080421430426091551))
```
Comment (nickheyer) [inline src/bot/discord_bot.py:434]: I assume this is where the commands are added to the server/channel/guild.

I store server IDs in a db. We could create a tree_sync function that runs in on_ready. Looping through each server_id in a db query - or even all server id's that this bot can currently see (ie: has been invited to)

File: src/bot/discord_bot.py (hunk 1 above)
Comment (Pecunia201) [reply src/bot/discord_bot.py:437]: Yes, it's just for debugging. Thought I removed that, my bad!

**Comment by nickheyer** [comment]
Based on new additions and refactors, in regards to slash commands. I am going to go ahead and close this PR so we can get the slash commands into the dev branch. If you'd like, please pull/fork/etc from the new dev branch after I merge if you want to keep working on it :)

**Pull request closed without merging**